applications/EmpireApplication/tests/sdof_static_solver/cl_ref.dat

```diff
@@ -0,0 +1 @@
+ 0.327805503865
```

Comment:
Instead of using a file to check one value you can check it directly in the test. Check the compressible potential flow application